sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/checks.sql

```diff
@@ -0,0 +1,32 @@
+#fail
+{{ is_unique(columns=["client_id"]) }}
+
+#fail
+ASSERT(
+  SELECT
+    COUNTIF(first_seen.client_id IS NULL)
+  FROM
+    `{{ project_id }}.{{ dataset_id }}.clients_daily_v6` daily
```

Comment:
Is this check meant to ensure all clients in the daily table have an entry in our table?

Reply:
Yup, and similarly for the new profile ping table